### PR TITLE
Add Custom Tags

### LINKED ISSUE #29: Sound tagging
Tag sounds with a label or labels.

### LINKED ISSUE #76: Add ability to sort sounds in ui
- Name
- Recently added

Depends on #75

### PR DESCRIPTION
**Pull request by stuckinab0x**
fix #29 
fix #76

### CODE REVIEW

File: projects/frontend/src/providers/SWRProvider.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { FC, PropsWithChildren } from 'react';
 import { SWRConfig, SWRConfiguration } from 'swr';
 
-const value: SWRConfiguration = { fetcher: (input, init) => fetch(input, init).then(x => x.json()) };
+const value: SWRConfiguration = { revalidateOnFocus: false, fetcher: (input, init) => fetch(input, init).then(x => x.json()) };
```
Comment (stuckinab0x) [inline projects/frontend/src/providers/SWRProvider.tsx:4]: I think the refresh token fails I was seeing miiiight have been from this sending requests in quick succession and discord rejecting a dupe or something after.

Technically this is our problem but I'll turn this off until it gets sorted, I can't think of what we do that needs to be constantly refreshing unless you want ALL users to get a new sound immediately when they tab in.

**Comment by stuckinab0x** [comment]
oh we need a migration too don't we whoopsie

**Review by UncleDave — CHANGES_REQUESTED** [review]
Looks good to meeeeeeeeeeeeeeeeeeeeee

**Comment by UncleDave** [comment]
When you fix the conflicts could you also merge dependabot's open branches into yours so we don't have to merge 6 PRs after this one?

**Review by UncleDave — APPROVED** [review]

**Pull request merged**